src/screenfull.js

```diff
@@ -109,7 +109,9 @@
 					resolve();
 				}.bind(this);
 
-				document[fn.exitFullscreen]();
+				if (Boolean(document[fn.fullscreenElement])) {
```

Comment:
`Boolean` is moot here.

Reply:
This fix is also not correct. The promise will never resolve if this statement is true.

Reply:
yeah, actually, I just get rid of the `Boolean`, because since the `fullScreenElement` function return `null` in case of falsy value. I fixed the thing, I think :)

> The built code should not be included. The code is built on new versions only.

Yes, sure, my bad.